### PR TITLE
fix(tabs): reduce overflow scroll button padding

### PR DESCRIPTION
**Pull request by zeroedin**
## What I did

1. Reduced padding on the overflow scroll buttons from `--rh-spacer-xl: 24px` to `--rh-spacer-lg: 16px`

## Testing Instructions

1. View [Deploy Preview](https://deploy-preview-956--red-hat-design-system.netlify.app/elements/tabs/demo/)

### CODE REVIEW

File: .changeset/neat-pumas-peel.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@rhds/elements": patch
+---
+
+`rh-tabs`: reduced padding on overflow scroll buttons
```
Comment (bennypowers) [inline .changeset/neat-pumas-peel.md:5]:
```suggestion
`<rh-tabs>`: reduced padding on overflow scroll buttons
```

**Review by bennypowers — CHANGES_REQUESTED** [review]

**Review by bennypowers — APPROVED** [review]

**Comment by zeroedin** [comment]
awaiting updated/finalized specs from @coreyvickery

**Comment by zeroedin** [comment]
Should also make sure this change is implemented in `rh-subnav` to ensure look and feel consistency.

**Comment by zeroedin** [comment]
updated xd spec: https://xd.adobe.com/view/e0eb07d2-71ac-498f-9d74-b872dd7c204b-b5b8/

**Pull request merged**